### DISCUSSION TITLE
2007/06/28/%E7%88%B6%E6%AF%8D%E4%B8%8E%E5%90%8C%E4%BC%B4%E5%AF%B9%E5%AD%A9%E5%AD%90%E7%9A%84%E5%BD%B1%E5%93%8D

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#170] [Announcements]
# 2007/06/28/%E7%88%B6%E6%AF%8D%E4%B8%8E%E5%90%8C%E4%BC%B4%E5%AF%B9%E5%AD%A9%E5%AD%90%E7%9A%84%E5%BD%B1%E5%93%8D

人是天性和教养（基因和环境影响）共同的结果。虽然基因很重要，但他是我们所不能改变的。而教养的影响使得我们有可能通过自己的努力，使自己或帮助别人取得更大成绩。所以，不论基因的影响有多大，作为一个个体，把精力关注到后天教养上，才会对自己更有好处。

https://blog.qizhen.xyz/2007/06/28/%E7%88%B6%E6%AF%8D%E4%B8%8E%E5%90%8C%E4%BC%B4%E5%AF%B9%E5%AD%A9%E5%AD%90%E7%9A%84%E5%BD%B1%E5%93%8D

**Comment by ruanqizhen** [comment]
之前博客下的留言：

竹贤
2007年06月29日 16:55 编辑
天性和教养肯定是决定个一人素质的最为基本的、极为重要的因素，但若仅仅看到这些就太简单了。其实，决定每个人的一生命运的有时就往往是一些所谓的小事，他们也许就是点燃导火索的一个小小的火花，但阴差阳错地改变了一个人的命运。有人从一件小事中悟出了人生的哲理，从此是浪子回头金不换；也有人是偶一失足，于是就酿成千古恨。
也正是因为人生的不可预测，才能够让永远我们憧憬美好的未来，寄托于美妙的理想，在人生的路上奋勇向前。

回复
Kin f.
2007年06月28日 21:51 编辑
挖塞
这么规范
这么专业
就是看不明白啊
